interfaces/sourcegen/src/sourcegen/headers/ct_auto.yaml

```diff
@@ -11,12 +11,8 @@ recipes:
   implements: version  # inconsistent API (preexisting)
 - name: getGitCommit
   implements: gitCommit  # inconsistent API (preexisting)
-- name: getCanteraError
-- name: setLogWriter
-- name: setLogCallback
-- name: writeLog
-  implements: writelog_direct  # inconsistent API (preexisting)
 - name: addCanteraDirectory
+  # TODO: Make API consistent, as signature changed compared to old CLib
```

Comment:
What is the change that needs to be made for consistency? Renaming this Clib function to be just "addDirectory"?